### PR TITLE
[ACA-3908] use different selectors that are valid for ADW as well

### PR DESCRIPTION
**Pull request by adinapitul**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
use selectors that are valid for both ACA and ADW

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: protractor.conf.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const SAVE_SCREENSHOT = process.env.SAVE_SCREENSHOT === 'true';
 const API_CONTENT_HOST = process.env.API_CONTENT_HOST || 'http://localhost:8080';
 const MAXINSTANCES = process.env.MAXINSTANCES || 1;
 const MAX_RETRIES = process.env.MAX_RETRIES || 1;
-const LOG_LEVEL = process.env.LOG_LEVEL || 'ERROR';
```
Comment (eromano) [inline protractor.conf.js:27]: better to not change this variable is the same in all he repo and documented

**Review by eromano — CHANGES_REQUESTED** [review]

File: protractor.conf.js (hunk 1 above)
Comment (adinapitul) [reply protractor.conf.js:27]: It's E2E_LOG_LEVEL in apps and adw and LOG_LEVEL in adf.... :)
I don't know who to be consistent with :D

Comment (adinapitul) [reply protractor.conf.js:27]: @eromano , please see my previous comment and let me know if you still think I should keep the name LOG_LEVEL.

Comment (eromano) [reply protractor.conf.js:27]: then can we cahnge it aso in ADF and change it also in the confluence page?https://alfresco.atlassian.net/wiki/spaces/AP/pages/471169/Running+e2e+tests

**Review by eromano — CHANGES_REQUESTED** [review]

**Review by eromano — APPROVED** [review]

**Pull request merged**